services/graph/pkg/identity/noop_school.go

```diff
@@ -0,0 +1,71 @@
+package identity
+
+import (
+	"context"
+	"net/url"
+
+	libregraph "github.com/owncloud/libre-graph-api-go"
+)
+
+// NOOP is a dummy EducationBackend, doing nothing
+type NOOP struct{}
+
+// CreateEducationSchool creates the supplied school in the identity backend.
+func (i *NOOP) CreateEducationSchool(ctx context.Context, school libregraph.EducationSchool) (*libregraph.EducationSchool, error) {
+	return nil, errNotImplemented
```

Comment:
NOOP struct should not return any errors (on all of these methods). Otherwise it is falsely named and should be called `ErrEducationBackend`. If NOOP struct is not used consider removing it

Reply:
Hm, true. I'll rename.